prefect_aws/workers/glue_job_worker.py

```diff
@@ -0,0 +1,156 @@
+import logging
```

Comment:
Can you add a module comment documenting how to use this worker? You can use the ECS worker [module comment](https://github.com/PrefectHQ/prefect-aws/blob/main/prefect_aws/workers/ecs_worker.py#L1-L46) as an example.

Reply:
Thanks, fix it
https://github.com/PrefectHQ/prefect-aws/pull/308/commits/68b02b8890c9f445c5e735d722560bca2e409de6